lib/ansible/modules/system/mount.py

```diff
@@ -57,18 +57,22 @@
     default: 0
   state:
     description:
-      - If C(mounted), the device will be actively mounted and appropriately
-        configured in I(fstab). If the mount point is not present, the mount
+      - If C(mount), the device will be actively mounted without changing
```

Comment:
The `state` value should always be in past tense. Please change it to `mounted_raw`.